### PR TITLE
Removing screenreader callout

### PR DESCRIPTION
**Pull request by emchap**
See convo here: https://sourcegraph.slack.com/archives/C0190QFK49X/p1631661825006400

### CODE REVIEW

File: doc/admin/config/settings.md [hunk 1]
```diff
@@ -8,10 +8,6 @@ Settings can be set at the global level (by site admins), the organization level
   <object data="settings-cascade.svg" type="image/svg+xml" style="width:80%;"></object>
 </div>
 
-<div class="text-center small">
-  Non-sighted users can view a <a href="settings-cascade">text-representation of this diagram.</a>
```
Comment (limitedmage) [inline doc/admin/config/settings.md:12]: We should also remove the page this links to

Comment (emchap) [reply doc/admin/config/settings.md:12]: image

I feel dumb—do you know if this is the markdown, or the svg we need to nuke? (Or both?)

Comment (limitedmage) [reply doc/admin/config/settings.md:12]: I think just the `.md` as the `.svg` is used inside `settings.md`.

Comment (emchap) [reply doc/admin/config/settings.md:12]: Thank you!

**Review by quinnkeast — APPROVED** [review]

**Comment by emchap** [comment]
@quinnkeast thank you!

**Pull request merged**